### DISCUSSION TITLE
[Idea] Introducing one-click deploy options

### DISCUSSION THREAD

**Discussion by kaiwalyakoparkar** [PrestaShop/PrestaShop#34368] [Ideas]
### The problem

The current readme has options to configure the app and self host but no fully managed or one-click deployment options present

### The solution

Elestio has been providing options of fully deploying and managing Prestashop application as shown [here](https://elest.io/open-source/prestashop). I think it would be a great idea if we can add it to official readme/documentation here.

### Additional context

:coll  In addition to this, if you are interested we provide collaboration opportunities with tools we support by revenue share upon addition of this method in docs. If you would like to collaborate, just shoot me an email at kaiwalya@elest.io :)

### Do you plan to work on this subject?

- [ ] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [ ] I'm willing to provide any wireframes or design assets required for this feature.
- [X] I'm willing to submit a Pull Request that implements this feature.
- [ ] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by MatShir** [comment]
We try to avoid commercial services and products in the documentation. I recommend you to share your solution here https://github.com/PrestaShop/PrestaShop/discussions/categories/showcase.
For now, only project members of the organization can highlight their services on dedicated pages.
